commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliAzureKeyVaultParameters.java

```diff
@@ -18,35 +18,43 @@
 
 public class PicoCliAzureKeyVaultParameters implements AzureKeyVaultParameters {
 
+  @Option(
+      names = {"--azure-vault-enabled"},
+      description =
+          "Set true if Web3signer should try and load all keys in the specified vault "
+              + "(Default: ${DEFAULT-VALUE})",
+      paramLabel = "<ENABLED>")
```

Comment:
thinking the paramLabel should be BOOL as that's what have done elsewhere

Reply:
done